src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -272,24 +272,19 @@ _parseDashArray(SvgLoaderData* loader, const char *str, SvgDash* dash)
 
 static string* _idFromUrl(const char* url)
 {
-    char tmp[50];
-    int i = 0;
-
     url = _skipSpace(url, nullptr);
     if ((*url) == '(') {
         ++url;
         url = _skipSpace(url, nullptr);
     }
 
+    if ((*url) == '\'' || (*url) == '\"') ++url;
     if ((*url) == '#') ++url;
 
-    while ((*url) != ')') {
-        tmp[i++] = *url;
-        ++url;
-    }
-    tmp[i] = '\0';
+    int i = 0;
+    while (url[i] > ' ' && url[i] != ')' && url[i] != '\'' && url[i] != '\"') ++i;
```

Comment:
I don't think it's possible to find a `"` sign in the url arg, since we are using `"` to determine the beginning and the end of a tag/value etc.
and `url[i] > ' '` - url can not contain spaces, what is the scenario for that?

Reply:
That's a fine catch with a quote sign. Will change it.
url must not contain any whitespace and `url[i] > ' '` should stop searching on '\0', space, tab, new line, etc

Reply:
if it contains a white space etc, doesn't in mean that it's invalid ?